### PR TITLE
fix(ui5-card): allow accessibleName to be used

### PR DESCRIPTION
**Pull request by GerganaKremenska**
accessible-name and accessible-name-ref can be used. 

FIXES: #3730

### CODE REVIEW

File: packages/main/src/Card.js [hunk 1]
```diff
@@ -116,8 +143,9 @@ class Card extends UI5Element {
 		return !!this.header.length;
 	}
 
-	get _ariaCardRoleDescription() {
-		return this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_CARD);
+	get _getAriaLabelledby() {
```
Comment (gmkv) [inline packages/main/src/Card.js:146]: better to name this method _getAriaLabel as it concerns the `aria-label` attribute

File: packages/main/src/Card.js [hunk 2]
```diff
@@ -47,6 +48,18 @@ const metadata = {
...
 	properties: /** @lends sap.ui.webcomponents.main.Card.prototype */ {
 
+		/**
+		 * Defines the accessible name of the component.
+         * <b>Note:</b> <code>accessibleName</code> is required for <ui5-card>, it provides unique name for the card region.
+		 *
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @public
+		 * @since 1.0.0-rc.15
```
Comment (fifoosid) [inline packages/main/src/Card.js:58]: rc16

File: packages/main/src/Card.js [hunk 3]
```diff
@@ -47,6 +48,18 @@ const metadata = {
 	},
 	properties: /** @lends sap.ui.webcomponents.main.Card.prototype */ {
 
+		/**
+		 * Defines the accessible name of the component.
+         * <b>Note:</b> <code>accessibleName</code> is required for <ui5-card>, it provides unique name for the card region.
```
Comment (fifoosid) [inline packages/main/src/Card.js]: is required for the component, it ...

**Review by fifoosid — CHANGES_REQUESTED** [review]
The title of the PR says that two properties are introduced, but as much as I can see, only one is added to the metadata

**Comment by GerganaKremenska** [comment]
> The title of the PR says that two properties are introduced, but as much as I can see, only one is added to the metadata

I forgot to update the title of the PR it is done now.

**Review by fifoosid — COMMENTED** [review]
Everything looks fine, but you should merge the master branch for the build to pass

**Review by fifoosid — APPROVED** [review]

File: packages/main/test/specs/Card.spec.js [hunk 4]
```diff
@@ -86,6 +84,17 @@ describe("CardHeader", () => {
...
+    before(async () => {
+        await browser.url(`http://localhost:${PORT}/test-resources/pages/Card.html`);
+    });
+
+    it("test accessibleName", async () => {
+        const card = await browser.$("#textCard").shadow$(".ui5-card-root");
 
+        assert.strictEqual(await card.getAttribute("aria-label"), "Card Internships",
+            "The aria-label is correctly when accessibleName is used.");
+    });
```
Comment (dimovpetar) [inline packages/main/test/specs/Card.spec.js:99]: Please indent with tabs here

File: packages/main/src/Card.js [hunk 5]
```diff
@@ -116,8 +129,9 @@ class Card extends UI5Element {
 		return !!this.header.length;
 	}
 
-	get _ariaCardRoleDescription() {
-		return this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_CARD);
+	get _getAriaLabel() {
+		const effectiveAriaLabel = getEffectiveAriaLabelText(this) ? ` ${getEffectiveAriaLabelText(this)}` : "";
```
Comment (dimovpetar) [inline packages/main/src/Card.js:133]: Avoid calling `getEffectiveAriaLabelText(this)` twice

File: packages/main/src/Card.js [hunk 6]
```diff
@@ -47,6 +48,18 @@ const metadata = {
 	},
 	properties: /** @lends sap.ui.webcomponents.main.Card.prototype */ {
 
+		/**
+		 * Defines the accessible name of the component.
+		 * <b>Note:</b> <code>accessibleName</code> is required for the component, it provides unique name for the card region.
```
Comment (dimovpetar) [inline packages/main/src/Card.js:53]: How about:
<b>Note:</b><code>accessibleName</code> is used as the name of the card region and should be unique.

**Review by dimovpetar — CHANGES_REQUESTED** [review]

**Review by dimovpetar — APPROVED** [review]

**Review by fifoosid — APPROVED** [review]

**Pull request merged**